config/jobs/kubernetes-sigs/image-builder/image-builder-presubmits.yaml

```diff
@@ -129,9 +129,8 @@ presubmits:
     labels:
       preset-dind-enabled: "true"
       preset-kind-volume-mounts: "true"
-    always_run: false
-    optional: true
     decorate: true
+    run_if_changed: 'images/capi/(Makefile|Dockerfile|\.dockerignore)|images/capi/hack/ensure-(ansible|packer|jq|azure-cli|goss).*'
```

Comment:
@codenrhoden given that we only have two data points on this job so far (https://prow.k8s.io/job-history/gs/kubernetes-jenkins/pr-logs/directory/pull-container-image-build) can we start by adding run_if_changed and keeping it optional and then switching it to required after ~ 1 week of stable data? Just being extra conservative given our recent CI flakes.